front/lib/api/assistant/pubsub.ts

```diff
@@ -79,6 +79,8 @@ export async function postUserMessageWithPubSub(
               return null;
           }
         }
+      } catch (e) {
+        logger.error({ error: e }, "Error Posting message");
```

Comment:
Unrelated: just to make it easier to read error that occurs in `postUserMessage`